addon/mixins/taggable-mixin.js

```diff
@@ -8,7 +8,7 @@ export default Ember.Mixin.create({
             Ember.A(currentTags);
```

Comment:
But you're right, I could simplify with `var currentTags = Ember.A(resource.get('tags'));`